Libraries/SDHC/Source/sdhc_lib.c

```diff
@@ -159,11 +159,20 @@ int MXC_SDHC_Lib_GetCSD(mxc_sdhc_csd_regs_t *csd)
 }
 
 /* ************************************************************************** */
-unsigned int MXC_SDHC_Lib_GetCapacity(mxc_sdhc_csd_regs_t* csd)
+unsigned long long MXC_SDHC_Lib_GetCapacity(mxc_sdhc_csd_regs_t* csd)
 {
     unsigned int size = csd->csd.c_size;
 
-    return (size*(512*1024));
+    return ((unsigned long long)size)*((unsigned long long)512*1024);
```

Comment:
FYI this may cause integer overflow with the FatFS library if it's casted to `DWORD`.  I fixed that overflow issue in https://github.com/Analog-Devices-MSDK/msdk/pull/720.

I think this change is ok since `MXC_SDHC_Lib_GetCapacity` is not used by FatFS directly or any existing examples.